wagtail_localize_rws_languagecloud/tests/test_models.py

```diff
@@ -76,7 +76,33 @@ def test_translations_ready(self):
         self.file.save()
         self.assertEqual(self.file.combined_status, "Translations ready for review")
 
-    def test_translations_published(self):
+    def test_translations_no_revision(self):
```

Comment:
suggestion: add a test for the translation.has_unpublished_changes (expecting "Translations ready for review")

Reply:
I think this is already covered by the existing test `test_translations_ready`